easyvvuq/analysis/sc_analysis.py

```diff
@@ -679,22 +652,76 @@ def get_moments(self, qoi):
         logging.debug('done')
         return mean_f, var_f
 
+    # def sc_expansion(self, samples, x):
```

Comment:
If it's commented out it should be removed. It is going to be preserved in the git history anyway.

Reply:
This was done by @wedeling and is not something I changed ... I would prefer not to remove the lines as I assume he had a good reason for keeping them ...

Reply:
I removed the commented subroutine